backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientResolver.java

```diff
@@ -33,7 +33,31 @@ public List<Person> getPatients(
       String namePrefixMatch,
       boolean includeArchivedFacilities) {
     return _ps.getPatients(
-        facilityId, pageNumber, pageSize, showDeleted, namePrefixMatch, includeArchivedFacilities);
```

Comment:
No, I think you're right - I will triple-check, but afaict the frontend never performs any `patients([...], showDeleted: true)` GraphQL queries. Leaving the _service code_ backwards compatible may be reasonable since there are definitely many invocations of `PersonService::getPatients` for deleted patients, but I think we can safely change what this resolver in returning in-place without issue.